MontyHallKata/Models/DoorsFactory.cs

```diff
@@ -0,0 +1,51 @@
+using MontyHallKata.Models.Entity;
+
+namespace MontyHallKata.Models
+{
+    public static class DoorsFactory
+    {
+        public static Door CreateWinningDoor()
+        {
+            var winningDoor = new Door
+            {
+                IsWinningDoor = true
+            };
+            return winningDoor;
+        }
+        public static Door CreateLosingDoor()
+        {
+            var winningDoor = new Door
+            {
+                IsWinningDoor = false
+            };
+            return winningDoor;
+        }
```

Comment:
Some code duplication here. Would it be possible to merge these two somehow? Or is there some other reason for keeping them separate?